### ISSUE TITLE
Unable to connect to my domain

### ISSUE THREAD

**Issue by FreemoX** [pierre-alain-b/rainloop-nextcloud#41]
Hello. First off, nice work on this app!

I'm attempting to set up the emails, so I added my domain with the proper IMAP/SMTP settings. 

My issue is, when I log in to the webmail server that my domain host is hosting, I have to use a username, so I can't simply log in with the full email address. 

For clearification:

Normally you would log in to services like Gmail with:
someone@gmail.com
MyVerySecretPassword123

But I have to log in to my domain email with:
myusername
MyAlsoVerySecretPassword321

The system works so that each user have to have their own user account, and then the actual email accounts are connected up to select user accounts.
I have no issues logging in to my email on the other email client on Nextcloud, because that one allows me to input the username and password for the actual account, as I can't simply log in with an email.

Would implementing a IMAP/SMTP username/password feature be possible with Rainloop?

**Comment by pierre-alain-b** [comment]
Hello,
I am not sure to understand fully the issue. When you set up your domain, have you ticket the option "use short login"?

![image](https://user-images.githubusercontent.com/386857/33739775-ab19dc3c-db9e-11e7-9093-2120f89596e7.png)

**Comment by FreemoX** [comment]
I have attempted every combination of the settings provided in the admin panel, as well in combinations with aliases. I didn't notice any difference when I toggled "Use short login" on either IMAP or SMTP.

As you can see on the image below, I can input the IMAP/SMTP username and password in the fields marked by the yellow highlight. The screenshot is from the "Mail" interface.
https://i.imgur.com/qutqpUY.png

But there are no option to input IMAP/SMTP username and/or passwords in Rainloop. Below is a screenshot of the settings I've added, but that's as far as I get. 
https://i.imgur.com/qBLNJSN.png

I might have misunderstood or missed something, but it seems like either Rainloop is unable to log in with IMAP/SMTP usernames, or my domain host is incompatible with the way Rainloop attempts to login (Domeneshop.no disables the ability for external servers to manage the email, but they allow client software like Gmail, outlook and apparently the "Mail" app in Nextcloud to access the email accounts through the IMAP/SMTP service.

**Comment by pierre-alain-b** [comment]
Rainloop is of course able to connect to IMAP/SMTP with username/password.

Now that you have created a domain, any user claiming an address from that domain on your NC instance will be able to connect.

Username and password are asked when launching Rainloop from your user account.

So the steps are:
I) set your domain (e.g. domain.tld, with imap.domain.tld and smtp.domain.tld) in the Rainloop admin panel
II) close the admin panel
III) open Rainloop as a user and you are asked for your email and password. Here input the address and IMAP password. Beware that if you have ticked the 'short login' box, only the non domain part of the email will be used as the IMAP username (which in your case is what you want if I get it right).
IV) that's it!

**Comment by FreemoX** [comment]
Thanks for the step by step, although that was how I figured it was done. But I'm not able to connect to my account with that method.

Here is how I log in to my domain host webmail (webmail.domeneshop.no)

1. Type in the username "mentaloutcastga10" into the username field.
2. Type in the password for the mentaloutcastga10 account.
3. I'm now logged in. 
4. Next, it loads the default email account connected to the user account "mentaloutcastga10", which is "no-reply@mentaloutcastgames.no".

If I want to connect an alternate email to the user account "mentaloutcastga10";

5. I create the email account "server-info@mentaloutcastgames.no. No password is used on the email accounts.
6. I connect the newly made "server-info@mentaloutcastgames.no" to the user account "mentaloutcastga10".
7. I can now log in to the user account "mentaloutcastga10", and I now have access to both "no-reply@mentaloutcastgames.no" and "server-info@mentaloutcastgames.no" from the same user account.

Note that the user account and email account are two different accounts. 

The email account is simply an address to send emails to. The best example I can come up with.

The email account is your house. It has an address which people can send mail to. But there are several people living in that house. The people are user accounts. There can be several people in one house, and some people have access to several houses. 

To sum up:

I can't log in with "mentaloutcastga10@mentaloutcastgames.no" as it is not an email address. "mentaloutcastga10" is only a user account that have access to the email account "no-reply@mentaloutcastgames.no"

I also can't log in with "no-reply@mentaloutcastgames.no", since it is not a user account, and have no password connected to it.

I **must** log in with "mentaloutcastga10" to have access to my email accounts, since domeneshop.no treat email accounts and user accounts differently.

**Comment by pierre-alain-b** [comment]
OK then I assume it is an incompatibility with the way your authentification works. Sorry.
